slack-moderator/content.go

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"sigs.k8s.io/slack-infra/slack"
+)
+
+func (h *handler) removeUserContent(interaction slackInteraction, duration time.Duration, targetUser string) (removedFiles, remainingFiles, removedMessages, remainingMessages int, err error) {
+	if duration > 48*time.Hour {
```

Comment:
is this a self-imposed limit?

Reply:
after reading the rest: yes.

can we pull this out somewhere so that it's declared in common with the options for how far back we allow?